apps/src/sites/studio/pages/components/FlexGroup.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import color from '../../../../util/color';
```

Comment:
The main downside to using `@cdo/apps` in more places is that static analysis and refactoring tools are less likely to understand what they mean (as that would require them to understand webpack module aliases)